packages/controls/jest.config.json

```diff
@@ -2,5 +2,6 @@
   "testRegex": "(\\./__tests__/.*|(\\.|/)(test|spec))\\.(types\\.)?[jt]sx?$",
   "transform": {
     "^.+\\.(t|j)sx?$": "@swc/jest"
-  }
+  },
+  "setupFilesAfterEnv": ["<rootDir>/src/jest-setup.ts"]
```

Comment:
Added a mock for `uuid`.

Reply:
Is it for readability or repeatability or both? Or some other reason?

Reply:
`uuid` is mocked for stable results across different runs.